### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by nasko222** [nasko222/the-minecraft-recreation-project#94] [Announcements]
Here I will post any updates I'm doing to the project.

**Comment by nasko222** [comment]
Today I updated b1.7 devs, the dev2 is the old dev1, and dev1 is the Hippoplatimus's piston mod but ported properly to b1.7 without modloader and updating the game also updates the pistons properly

The reason i did this is because I believe Jeb originally started from the piston mod. He once mentioned something like "16... I changed it to 12 because I think its too much in the mod" (Talking about push length)

Speculative remake, you might skip, Anyways seeya.

**Comment by nasko222** [comment]
Today I've pushed a new Indev dev build (Discovered by minecraft12) in-20100209, It's a 207-2 base but with Notch skin and armor. 

https://github.com/nasko222/the-minecraft-recreation-project/issues/97 for more info

**Comment by nasko222** [comment]
Today I changed the names of the May 14 preclassic speculative builds that were originally given to me by SeaGlowingPro.

I had Vector Graphics tell me rough timestamps of when notch was active and its better to have that than nothing. Hashes remain the same

**Comment by nasko222** [comment]
I've updated the 1231-1 indev with proper base one and I properly removed dropping as keybind and other inventory ways to drop. There's also dev build now before dropping was added, the order is -1, dev, -2

**Comment by nasko222** [comment]
I renamed 125-dev to 126

**Comment by nasko222** [comment]
I forgot to mention that I added 12w03a snapshot devs, The tree gen is almost perfect

But i am not here for that, I am here to announce that I moved the very speculative builds to this repo https://github.com/nasko222/highly-speculative-mc-builds and removed them from the TMRP hash index.

i might remove few more, I am questioning about 2a-dev and inf 308 but for these we have actually word by notch xd

**Comment by nasko222** [comment]
Updated the second dev of b1.6, (TB2) Sky dimension test, added flying with the J hotkey like in the other pre b1.8 devs

**Comment by nasko222** [comment]
updated 19a, tweaked the glass texture opacity a bit (a lot)

**Comment by nasko222** [comment]
c0.24-14 has been updated with small change, arrows are now fired with J instead of TAB

**Comment by nasko222** [comment]
Added water damage to 0.24 dev6

**Comment by minecraft12y** [comment]
Forgot to update this, uploaded a1.1.1-dev. It's basically a1.1.0 but with volume and sensitivity sliders. It was talked about briefly on The Word of Notch on September 17.

**Comment by nasko222** [comment]
I havent been keeping with updates lately so here they are:

B1.6 dev order, b1.5_02-dev is the pre-map test without crafting recipe, b1.6-dev is the sky dimension. The test builds are OpenGL Related
PC-131648 and 131655 had their lighting tweaked properly to match the video
Alpha 1.2.0 dev order has been updated, dev3 is the portal test, dev4 is the early biome test and dev5 is the late biome test
b1.9 pre4 dev2 had the dragon wing animation removed

**Comment by nasko222** [comment]
oh yea i reverted the revert for a fix for the render bug for 152244,

im not touching pc again, i am fed up from it, too many speculative and random shit, and yeah its most likely not 152244 but IDC im leaving it as is

BTW i forgot to mention about the alpha devs, i forgot that i codeed the passive pigman in the earlier devs, and left it there, thats half speculative but i dont have anything better, might as well have it as easter egg, meh

**Comment by nasko222** [comment]
last thing for tonight (HOPEFULLY) I fixed the leaves folliage for the early alpha 1.2.0 devs

**Comment by nasko222** [comment]
alright then maybe not...


well i removed the "Unlicensed copy" screen for b1.5_02, i somehow did it in the dev but forgot for this one

**Comment by nasko222** [comment]
122 and 124-1 updated

both:
TNT now deals less damage like in in-110, also it destroys water and lava like in 110

122:
The items no longer spin

**Comment by nasko222** [comment]
Todays updates:

c0.0.14a_04, 06 and 07 - better mouse issue replication (if you downloaded in the past few hours download again, i broke them really bad for a second)
Removed a1.0.1-dev
Re-added a1.0.6-dev, this time with a1.0.5_01 base, made by minecraft12
mega accurate dragon texture for b1.9-pre4-dev5

that should be all for now

**Comment by nasko222** [comment]
Today's updates are enchanting table themed :D 

new 1.8 snapshot dev, The enchantment table uses gold instead of lapis
b1.9 pre3 dev2 has been tweaked so you can use B to access creative and jump to get experience, due to more screenshots being shown and it looks like they had easy access to those

**Comment by nasko222** [comment]
Beta updates:
- b1.7 dev and pre, Brand new ones, the pre one is 100% code recreation from the mcdev b1.7 pre. and the b1.7 dev uses the pre as a base while properly implementing the dev build piston behavior (and few other tweaks)
- b1.6 preview has been renamed to b1.6 trailer to keep the consistant naming scheme
- Removed all b1.8 devs and they have been unhashed, New b1.8 devs are cooking and will be coming soon, The unhashed b1.8 dev files can still be downloaded from this snapshot of the repo https://github.com/nasko222/the-minecraft-recreation-project/tree/51784a4a25b14950883dfd6efeedaff0e1de3856

**Comment by nasko222** [comment]
Classic updates.

0.24 1,2,3,4 - reverted thread sleep 5 to avoid issues on newer pcs
0.0.14a 04,05,06,07 (removed times) and uses new mouse issue code thats less apparent, but wont cause unplayability

**Comment by nasko222** [comment]
While working on The hybrid, Staggo showed me how to change world height in a very easy way, no porting anvil needed or stuff like that.

I took myself 5 minutes and went back and recreated the 227 dev 256 height test. IT WAS ONE VARIABLE THANK YOU NOTCH

whatever its up now ¯\_(ツ)_/¯

**Comment by nasko222** [comment]
new jar xd 0.27 (plain)

comes after 0.28 dev2, its a third dev build for 0.28 for creative tests

so order is:

0.27_st
0.28-dev
0.28-dev2
0.27
0.28
0.28_01

**Comment by nasko222** [comment]
107 update, Mipmapping issue that method and sith pointed out

**Comment by nasko222** [comment]
Updated again, fixed a bug where it crashes on AMD

**Comment by nasko222** [comment]
forgot to update this yesterday, inf 628 has been added, dev build for coral test and also updated 12w03a-dev with the jungle trees issues mentioned in https://github.com/nasko222/the-minecraft-recreation-project/issues/178

**Comment by nasko222** [comment]
(Reposted from main topic)

Hey everyone, our TMRP team has made a public TMRP (And The Hybrid) discord server for easier discussions about the project, sharing your ideas/feedback and other stuff, the server is still new and if this idea catches on it will be moderated properly in the future

Invite link - https://discord.gg/jQxU7c3k4X

**Comment by nasko222** [comment]
TMRP update in 2025 lol Flyby mode remakes done https://github.com/nasko222/the-minecraft-recreation-project/issues/182

**Comment by nasko222** [comment]
Fixed 0.26_st buttons issue https://github.com/nasko222/the-minecraft-recreation-project/issues/186#issuecomment-2679703265

**Comment by nasko222** [comment]
Version label changed in b1.7-dev to b1.6.6

**Comment by nasko222** [comment]
Forgot to mention about 0.24 1-11 fixed outline issue

Also new updates from today, Fixed an oversight with b1.5_02 map item and made b1.7-dev pistons hitbox bigger like in the screenshot.

**Comment by nasko222** [comment]
Updated b1.7-dev, TNT now acts like in b1.6.6 but with Shears you can defuse it - https://x.com/jeb_/status/76628452815540225

**Comment by nasko222** [comment]
Fixed https://github.com/nasko222/the-minecraft-recreation-project/issues/193

**Comment by nasko222** [comment]
New remake - https://github.com/nasko222/the-minecraft-recreation-project/issues/195

**Comment by nasko222** [comment]
Fixed issues with b1.2-dev2 and the snow generation is more accurate around cold areas

**Comment by nasko222** [comment]
Updates to b1.5 devs

Achievement menu now replicates the bug shown in the vid
Rain and thunder are tied up together like in the vid
Rain and thunder now only drop one light level (dev2)
Thunder properly removed (dev1)
Removed some statistics that werent added

Rain sounds changed to match the ones in the video
To clear the confusion with the new sound branch of the repo, this is the first time i had to add custom sounds to a recreation, credit to James for the rain sounds.

**Comment by nasko222** [comment]
Bug with "Achievement get" popup, sorry, reuploaded again

**Comment by nasko222** [comment]
Indev 124-1 was updated, the chest code was incorrect and now the correct chest code is used. To create funny chests place a chest between two chests

Chest word counter: 6

**Comment by nasko222** [comment]
Beta 1.9 changes

b1.9 beta 2 - Seperated to two versions, Recreated the funny notch joke with "Ultra hardcore" mode
b1.9 pre3 dev1-3 are now renamed to 2-4 to make space for dev1 which is "Pie chart test" displaying different pie chart
b1.9 pre3 prereupload 2 removed as the wiki is wrong and theres no debug code, its identical to the prereupload 1 which is found
b1.9 pre4 devs are temporary removed as we've been working on new much more accurate dragon models

**Comment by nasko222** [comment]
Brought back dev4 and dev5 of 14a (Snowy stuff)

**Comment by nasko222** [comment]
111-1 new build, full screen prereupload, F11 issues, can't toggle in menus and has the weird gray bar
111-2 - the old 111
111-114 - fixed a bug where the mouse disappears after full screen toggle

**Comment by nasko222** [comment]
Renamed 108 devs

20100108-dev becomes 20100108 as its the only dev build now
20100108-dev2 becomes 20100109 dev as the image is from 9th january

**Comment by nasko222** [comment]
New build, 128-1 - No crafting recipes and the menu should do nothing

**Comment by nasko222** [comment]
19a - glass made a little bit more transparent and removed weird pixel

**Comment by nasko222** [comment]
Removed early indevs, and opened a new branch on the repo
https://github.com/nasko222/the-minecraft-recreation-project/tree/testing
For testing new remakes, will make a new chat for that

**Comment by nasko222** [comment]
Early indevs updated

All - now use 1223-2 base
1207-1219-dev - Now uses the correct MD3 engine
1207 - All animations are made as accurately as currently possible to the videos with all clipping glitches
1219-dev1 - finite water, and tnt debug to check floodfill
1219-dev2 - shadows and inventory artifacts, preparing for dynamic lighting
1219-dev3 - dynamic lighting, ranas now static, light emitting and reflections dont work yet
1222-dev - light emitting and reflections now work, issues with half slabs
1222-dev2 - same build but in creative, ranas dont spawn, you can die in creative
1223-1 - no significant changes since last time, just some code cleanup

**Comment by nasko222** [comment]
I did a little bit of renaming for classic, indev and infdev, sorry for those who kept consistency with the archives

**Comment by nasko222** [comment]
New build, 218-1, No third person yet

**Comment by nasko222** [comment]
Small update to MD3 test that I forgot, the water texture no longer animates on the sides cause that was added on 19 december

**Comment by nasko222** [comment]
616-2 now actually works as intended (breaks level.dat)

Its code accurate cause the previewer in that version doesnt check for "canSave" and because theres no player it deletes the inventory

**Comment by nasko222** [comment]
211 updates
both - fixed player skin
211 - instead of having incorrect time set now it gradually breaks down as lighting updates and the lighting stops and causes mild lag (should be more accurate) loading a level fixes it but creating a new one doesnt (as people in irc mention)

131 updates

-1 - Initial
-2 - Fixes a bug in -1 where moving tool in inventory resets durability but whoops notch sucks and you spawn with wooden pickaxe and chests
-3 - Fixed spawning with wooden pickaxe and chests
-4 - Loading levels has been refined by using os dependent window instead of Java which fixes the Back to game button
-5 - Damage model update (Found)

Damage model prior:
All tools - deal 2 damage
Wooden sword - deal 4
Stone sword - deal 5
Gold sword - deal 6
Iron sword - deal 7
Diamond sword - deal 20

**Comment by nasko222** [comment]
Fixed rookie mistake in the 131s where sword durability is 9999 xdddddddddddd

**Comment by nasko222** [comment]
new stuff
b1.3dev was renamed to dev2 cause dev1 is now crying obsidian test (just for the funny)
Also in MD3 test, theres now MEGA secret feature :)

**Comment by nasko222** [comment]
MD3 test secret update was reverted because for whatever reason only MultiMC can load it and no Betacraft

**Comment by nasko222** [comment]
crying obsidian b updated again

**Comment by nasko222** [comment]
MD3 test updated, fixed missing md3 animations and lowered resolution as notch intended

**Comment by nasko222** [comment]
new remakes
129-2333 , 130 base but lacks few stuff
601-605 devs, brand new, lots of cool shit!!11

**Comment by nasko222** [comment]
Today's updates:

113, 113c, 114 - Fixed double dropping
122 - removed bow and arrow sounds
122, 124-1 - Removed double dropping
124-1 - You can now drop full stacks of items
0.25 st - renamed and mob ai tweaked so they can now chase dead entities
0.24 st and st 01- Block breaking one side bug now should be code accurate (And shows different on AMD)
(this is still to apply in 0.24 devs, more on that https://github.com/nasko222/the-minecraft-recreation-project/issues/207)
Christo18 is now a contributor to the project

**Comment by nasko222** [comment]
I forgot to mention the updates from last week (excluding stuff from /testing/)

0.24-8 now uses new better texture by BlueStaggo (this one will be updated again along all other 0.24s however)
0.26 all are updated with better timestamp of their features
0.27 was renamed to 0.28-dev3
brand new in-206 devs

Note: this update log might become obsolete in the future, our team plans to make a Wiki for TMRP that also fixes blatant and massive flaws the normal mc wiki has, We actually need wiki writers and helpers so if anyone is interested, a channel #wiki will be shortly created in the TMRP Discord server to begin that part of the project

**Comment by nasko222** [comment]
The Minecraft Recreation Project is now featured on the Minecraft Discontinued Features Wiki! (Under java mods)

https://mcdf.wiki.gg/wiki/Minecraft_Discontinued_Features_Wiki:Resources#Unofficial_Minecraft_Java_Mods

(Special thanks to <@667518806616440853> and the other MCDF wiki admins!

**Comment by nasko222** [comment]
Todays updates
326 - now spawns with items and NBT fully removed from code
325-1 - new remake - has the flower decay issue (mostly noticable during night time)
new versions for /testing/

**Comment by nasko222** [comment]
Today updates (Moved from testing)

204-206:
204-1 - new (re-)remake - uses 202 base, has the broken pathfinding and mob issues
204-2 - new (re-)remake - uses 202 base, same as last one but hostile
(in both of them the mobs save on file)
204-dev (previously known as 205-dev) - giant test
205-1 - uses 206 base, with the new performance improvements but has moonwalking bug
205-2 fixes moonwalking
206-1 fixes laggy text
206-dev - adds culling

renamed:
206-dev2 (prev dev1) - farming test 1
206-dev3 (prev dev2) - farming test 1

other:
209-dev - now has the new spawning
211-dev (new one) - 1 min daylight cycle
211-dev2 (previously known as dev1) - 5 min daylight cycle
211 - new way to reproduce the bug, now the lag spike gradually increases every time light update fails to happen

**Comment by nasko222** [comment]
two new remakes again

1225-dev - merry christmas update (notch adds isometric screenshots) (this marks it the only version with isometric screenshots and functional creative mode with block selector)
104-1 - sound engine internal crash bug (slient mode)

**Comment by nasko222** [comment]
Recent updates:

Classic: 22a all have the 22a_05 lava speed instead of 21a now (However their sound issues are still artificial)
Classic: Renamed few stuff
Testing: 113 and 113c, slow water depending on byte buffer issues (code accurate?)
Testing: 122 and 124-1, brand new remakes with IRC log proof
Testing: 131648, SeaGlowingPro's 131655 but with 8x8 chunks
Testing: 152225 with full screen and with deteriorate bug
Testing: 12a dev2 and dev3 using 12a_03 base by SeaGlowingPro ported by me and fixed few stuff with water
Testing: 13a dev2 and dev3, fixed first generation world sizes
Testing: 1a-3a new remakes using the last preclassic base with applet added (Mouse issues are artificial)

something else i forgot lolzerssss GO TEST THE REMAKES or 912784146723674932875 years of bad luck

**Comment by nasko222** [comment]
Yesterday's updates were b1.2 color dyes pallete and 12a-dev2 water border texture

Today in testing we have two Indevs, 207-2 (pre reupload) - Both Glass and Slabs should have light going through (and rain)
129-2333 - This one is not new, I just enabled VSync in it
